GDPR-Phase2-Java/src/main/java/com/gdpr/phase2/model/SortedVector.java

```diff
@@ -0,0 +1,39 @@
+package com.gdpr.phase2.model;
+
+import java.util.*;
+
+
+/**
+ * @Description
+ * Its a java replacement of the "Vector" class defined in the Typescript code of Chris's Library
+ */
+public class SortedVector {
+    public int bitLength;
+    public SortedSet<Integer> set;
+
+    public SortedVector(int bitLength, SortedSet<Integer> set) {
```

Comment:
Ah so much nicer to have a `SortedSet` in Java – JavaScript doesn't have that.

Reply:
We already had SortedSet in java so i just used it along with bitlength field as a replacement to your vector class in Typescript. Vector is already taken, so it creates confusion. So i renamed it to SortedVector class.

Reply:
Cool!